src/library/Tabs/TabList.js

```diff
@@ -0,0 +1,356 @@
+/* @flow */
+import React, { Children, Component } from 'react';
+import debounce from 'lodash.debounce';
+import EventListener from '../EventListener';
+import { createStyledComponent, pxToEm } from '../styles';
+import { createThemedComponent } from '../themes';
+import Button from '../Button';
+import IconChevronLeft from '../Icon/IconChevronLeft';
+import IconChevronRight from '../Icon/IconChevronRight';
+import IconExpandLess from '../Icon/IconExpandLess';
+import IconExpandMore from '../Icon/IconExpandMore';
+import { OverflowContainerWithShadows } from '../OverflowContainer';
+import { componentTheme as tabPanelComponentTheme } from './TabPanel';
+
+type Props = {
+  /** Horizontal or vertical alignment of TabList children */
+  align?: 'start' | 'center' | 'end' | 'justify',
+  /** Accessible label for TabList */
+  'aria-label'?: string,
+  /** Content of TabList */
+  children?: React$Node,
+  /** Maximum height of parent, e.g. Tabs */
+  maxHeight?: number | string,
+  /** Called when the selection is incremented to the next or previous item, e.g. Tab */
+  onIncrement?: (direction: string, event: SyntheticEvent<*>) => void,
+  /** Position of TabList in relation to related content, e.g. TabPanel */
+  position?: 'bottom' | 'start' | 'end' | 'top',
+  /** Accessibility role of TabList */
+  role?: string,
```

Comment:
Does the `TabList` role of `tablist` ever change?  If not, then can it just be hardcoded in `TabList` and the prop removed?

Reply:
It doesn't yet, but we built TabList to be easily used for purposes beyond Tabs (at which point we'll change the name).